spec/support/cluster_tools.rb

```diff
@@ -121,7 +121,7 @@ def step_up(address)
       rescue Mongo::Error::OperationFailure => e
         # Election failed. (125)
         if e.code == 125
-          # Possible reason is the node we are trying to elect has blacklisted
+          # Possible reason is the node we are trying to elect has denylisted
```

Comment:
I suppose `deny-listed` here would be good enough. "Blacklisted" here means a general prohibition rather than referring to a particular blacklist.